client/ngapp/views/nettests/dns-consistency.html

```diff
@@ -16,11 +16,11 @@
             <h3>What does this test do?</h3>
 
             <p>
-            This test compares the DNS query results from a known good DNS resolver and one that is to be tested for tampering.
+            This test compares the DNS query results from a DNS resolver which is considered to be reliable with one that is tested for tampering.
 
             </p><p>
-            The domain name system (DNS) is what is responsible for transforming a host name (e.g. torproject.org) into an IP address (e.g. 38.229.72.16). ISPs, amongst others, run DNS resolvers which map IP addresses to host names. In certain circumstances, ISPs map the wrong IP addresses to the wrong host names. This is a form of tampering, which OONI can detect by running its DNS consistency test, which compares the IP address of a given host name with the IP address allocated to that website by a provider. This test compares the IP address of a given host name with those provided by the Google DNS resolver, which we assume to not be tampered with. In short, when ISPs tamper with DNS answers, they replace them with something else. This results in being redirected to other websites or in failing to connect to the correct website. This is a form of censorship and network tampering.
-
+            The domain name system (DNS) is what is responsible for transforming a host name (e.g. torproject.org) into an IP address (e.g. 38.229.72.16). ISPs, amongst others, run DNS resolvers which allocate IP addresses to host names. In certain circumstances though, ISPs allocate the wrong IP addresses to the wrong host names. This is a form of tampering, which OONI can detect by running its DNS consistency test.</p>
```

Comment:
There is a problem with the terminology being used here: 
"run DNS resolvers which **allocate** IP addresses to host names" I think using the term **allocate** in the context of networking can be misleading as it usually is used to mean something quite different.

I would suggest replacing allocating with **map** instead as that is less ambiguous.